source/witch/witch.js

```diff
@@ -67,11 +67,11 @@ function respond(event, context, responseStatus, responseData, physicalResourceI
     var https = require("https");
     var url = require("url");
  
-    var parsedUrl = url.parse(event.ResponseURL);
+    var parsedUrl = new url.URL(event.ResponseURL);
```

Comment:
You could use destructuring here to make this more terse:

```js
var {pathname, search} = new url.URL(event.ResponseURL);
// ...
  path: pathname + search
// ....  
  ```

Reply:
You're right, I like it better. I am also not sure of how 'idiomatic' `new ur.URL(...)` is. I am by no mean a Node.js expert so I tried to change as little code as possible not to break anything. I see this PR is merged but maybe you could propose a subsequent one?